group_by.py

```diff
@@ -0,0 +1,18 @@
+from typing import Callable, Iterable
+
+
+def group_by(function: Callable, iterable: Iterable) -> dict:
+    """
+    :param function: Function run on all values in iterable.
+    :param iterable: Arguments for function.
+    :return: A dictionary with function(value) as a key and all values (with this key)
+    as a list.
+    """
+    ret = {}
```

Comment:
Don't use abbreviations in variable names -
https://softwareengineering.stackexchange.com/questions/72593/how-to-abbreviate-variable-names